saleor/webhook/payloads.py

```diff
@@ -208,6 +210,45 @@ def generate_product_payload(product: "Product"):
     return product_payload
 
 
+def generate_fulfillment_lines_payload(fulfillment: Fulfillment):
+    serializer = PayloadSerializer()
+    line_fields = ("quantity",)
+
+    return serializer.serialize(
+        fulfillment,
+        fields=line_fields,
+        extra_dict_data={
+            "weight": (lambda fl: fl.order_line.variant.get_weight().oz),
+            "product_type": (
+                lambda fl: fl.order_line.variant.product.product_type.name
```

Comment:
Did you test this code for database performance? It requires a few joins so I would just make sure that we have proper prefetching where it's necessary.

Reply:
I've already added prefetch here.  5 queries vs 89